core-backend/common/src/runtime.rs

```diff
@@ -20,9 +20,25 @@
 
 use crate::{
     memory::{MemoryAccessRecorder, MemoryOwner},
-    BackendExternalities, BackendState, TerminationReason,
+    BackendExternalities, BackendExternalitiesError, BackendState, TerminationReason,
 };
 use gear_core::{costs::RuntimeCosts, pages::WasmPage};
+use gear_core_errors::ExtError as FallibleExtError;
+
+#[derive(Debug, Clone, derive_more::From)]
```

Comment:
Also better to remove this derive_mode::From, because this makes code incredibly misunderstandable. Even using IDE does not help to see all deps between errors

Reply:
Even more may be better to remove `From` for some other errors also, so code with explicit errors names will be bigger but much more human friendly.

Reply:
> Even more may be better to remove From for some other errors also

It's already done in other refactoring PRs